docs/how-to/create-custom-image.md

```diff
@@ -44,4 +44,185 @@ Maven will use _maven-dependency-plugin_ to download the AMP from Maven reposito
+
+```bash
+docker push customrepository/alfresco-custom-image:customTag
+```
+
+*** Note: *** It is not necessary to build/have amps locally and copy them to the new docker image, if the .amp can be downloaded via network (FTP/HTTP/HTTPS), COPY can be replaced by CURL/WGET commands.
+
+
+### Amps that require additional configurations (advanced).
+
+***Note:*** We are going to use alfresco-saml-distribution in order to install Alfresco-SAML-Module(testing purposes only! it is recommended to use a proxy for SSL).
+### Steps:
+* We are going to use the following folder and file structure:
+
+```bash
+alfresco-custom-image                            (Dir)
+│
+└───Repository                                   (Dir)
+│   │
+│   └───amp                                      (Dir)
+│   │   │
+│   │   └alfresco-saml-repo-*.amp               (File)
+│   │
+│   └───keystore                                 (Dir)
+│   │    │
+│   │    └ssl.keystore                          (File)
+│   │    └ssl.truststore                        (File)
+│   │    └saml.keystore                         (File)
+│   │    └saml-keystore-password.properties     (File)
+│   │
+│   └───Dockerfile                              (File)
+│
+└───Share                                        (Dir)
+    │
+    └───amp_share                                (Dir)
+    │   │
+    │   └alfresco-saml-share-*.amp              (File)
+    │
+    └───keystore                                 (Dir)
+    │    │
+    │    └ssl.keystore                          (File)
+    │    └ssl.truststore                        (File)
+    └share-config-custom.xml                    (File)
+    │
+    └───Dockerfile                              (File)
+```
+* Generate keystore and truststore required for ssl and SAML. Additional information can be found under [alfresco documentation](https://docs.alfresco.com/).
+
+* Append the content of "share-config-custom.xml" to the one used in the alfresco-share docker image (the content that needs to be added is to big for the use of a SED command).
+
+* Add files to match the above hierarchy.
+
+* Create Dockerfile for custom-alfresco-repository image:
+
+```
+### We are going to apply amp(s) on the latest enterprise repository image.
+FROM alfresco/alfresco-content-repository
+
+### Copy the keystore build context to the appropriate location for your application server
+COPY keystore /usr/local/tomcat/keystore
+
+### Copy the amps from build context to the appropriate location for your application server
+COPY amps /usr/local/tomcat/amps
+
+
+### Install amps on alfresco
+RUN java -jar /usr/local/tomcat/alfresco-mmt/alfresco-mmt*.jar install \
+              /usr/local/tomcat/amps/alfresco-saml-repo-1.1.0-*.amp /usr/local/tomcat/webapps/alfresco -nobackup -force
+
+### Enable SSL by adding the proper Connector to server.xml (using a very hard to understand SED command)
```

Comment:
This may be appropriate for running the image locally or via Docker compose, but wouldn't we want to recommend SSL termination at the K8s ingress, not inside the container?

Reply:
There is a note at line 120, on which I try to reflect that:
```
***Note:*** We are going to use alfresco-saml-distribution in order to install Alfresco-SAML-Module(testing purposes only! it is recommended to use a proxy for SSL).
```
I would other comments to suggest the use of K8s ingress.